### ISSUE TITLE
[Question] Selecting in UI10 record doesn't tie with related list

### ISSUE THREAD

**Issue by waran70** [YetiForceCompany/YetiForceCRM#11932]
Related module have UI type 10 leading to parent records list. If I chose here parent, the related record will not show in Parent module related tab. I need to select record in related tab manually so it shows on the list. How to triger in this point inserting relevant record in vtiger_crmentity and vtiger_crmentityrel? Actually it would be pretty cool to make this working in core code

**Comment by rskrzypczak** [comment]
You probably have a wrong relation set between modules.
The correct relation type is added together with the reference field (uitype10).
You are not able to add this relation from the panel (Software configuration\Standard modules\Modules – Relations)

**Comment by waran70** [comment]
Well it is UI type 10.. I can select records, but they not showing in related list. As I said, this kind of manual relation could eventually be supported by code which will insert related records to the tables vtiger_crmentity and vtiger_crmentityrel. If I add manually such relation it ignoring related list. Having field type UI 10 in vtiger_field is not enough to support automated creation of related record.

**Comment by rskrzypczak** [comment]
Like I said, the UItype 10 reference fields have a 1:M relation and the information about the related records is not stored in the vtiger_crmentityrel table and will never be.
So if you added the relation manually from the panel  you created the relation type M:M, however, it has nothing to do with the UItype 10 reference field.
Adding a second relation to the same module could have caused a fair amount of chaos and you can have different related records displayed in different places.
Check what relationships you have for this module in the vtiger_relatedlists table.

To sum up:
The proper relationship (1:M) is added together with the UItype 10 field.
The configuration panel gives you the option to add only M:M relations, and manipulating relationships in this panel is risky and should be done consciously, because some relations cannot be restored after they’re deleted, so this option is only available for developers.

**Comment by JohnMaverick** [comment]
Why when I add Project to Project Task using reference field, this task showing in Project in related list of Project Tasks? I think something here has been misunderstood?

**Comment by JohnMaverick** [comment]
In vtiger this was easy by adding in vtiger_relatedlists "relationfieldid" but it has been removed in yetiforce.
